database.go

```diff
@@ -1065,7 +1065,7 @@ func (db *datastore) GetPostsCount(c *CollectionObj, includeFuture bool) {
 // GetPosts retrieves all standard (non-pinned) posts for the given Collection.
```

Comment:
Just need to update this comment to reflect the option of including pinned posts.

Reply:
I've updated the docstring as requested. :)